application/base_script.sh

```diff
@@ -0,0 +1,40 @@
+xcp_d \
+    --participant_label 01
+    -t rest \
+    --bids-filter-file filter.json \
+    --combineruns \
+    --cifti \
```

Comment:
Does this exclude niftis?

Reply:
Yes. With the `--cifti` flag, only CIFTIs will be postprocessed. If we want to postprocess both CIFTIs and NIfTIs, we'll need to run XCP-D twice.